src/content/structured/get-started/migrating-to-v3.mdx

```diff
@@ -0,0 +1,152 @@
+---
+path: "/get-started/migrating-to-v3"
+
+navPriority: 4
+
+date: "2025-02-13"
+
+title: "Migrating to V3"
+
+subTitle: "This guide is intended to help you migrate from ICDS 2.x components to ICDS 3.x components."
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/get-started/migrating-to-v3.mdx"
+---
+
+## Introduction
+
+<p>
+  ICDS V3 introduces one of our mostly highly anticipated features, dark mode,
+  as well as general improvements. This is a comprehensive guide on how to
+  upgrade your V2 app to V3. For a full list of the changes made in V3, please
+  read the{" "}
+  <ic-link
+    target="_blank"
+    href="https://github.com/mi6/ic-ui-kit/blob/develop/packages/web-components/CHANGELOG.md"
+    rel="noreferrer noopener"
+  >
+    changelog
+  </ic-link>
+  .
+</p>
+
+## Theming and dark mode
+
+With V3's arrival comes an improved `ic-theme` component. This component was initially responsible for managing the theme of the UI Kit. In addition to the existing functionality, you can now use the `ic-theme` component to set whether a component responds to changes in the system's light and dark theme or whether it always appears in a particular mode. Guidance on implementing the `ic-theme` package can be found in the [custom theme guidance](/get-started/development/custom-theme).
+
+## Codemod
+
+To aid your migration from ICDS V2 to V3, we have created a codemod to automate the process of migrating your code. A codemod is a tool that can be used to make large-scale changes to codebases quickly and efficiently. Our ICDS V2 to V3 codemod is the simplest and easiest way to migrate your codebase to the latest version of the UI Kit.
+
+### How it works
+
+Given a directory, it will scan over files and find any relevant ICDS components that require a change either by component name or props used.
+
+### How to run
+
+<p>
+  This package works as an executable requiring a directory and optional test
+  boolean argument to help migrate your tests from ICDS V2 to ICDS V3. Further
+  information and examples can be found in the{" "}
+  <ic-link
+    target="_blank"
+    href="https://github.com/mi6/ic-ui-kit/blob/v3.0.0/main/packages/codemod/README.md"
+    rel="noreferrer noopener"
+  >
+    codemod package README
+  </ic-link>
+  .
+</p>
+
+```console
+- npm @ukic/react@latest
```

Comment:
Should this be `npm i ...` or have I misunderstood and its doing something different?